### DISCUSSION TITLE
Albums - Lose all content - Both Memories and Photos and all Face recognition

### DISCUSSION THREAD

**Discussion by ebildebil** [nextcloud/all-in-one#5242] [Questions] [answered]
Hi @szaimen 

I have a dedicated server, on which I run Nextcloud AIO and Memories. In general all works well.

We create a lot of albums in Memories.
All our media is stored on a 2 different NAS, and the folder(s) is mounted into Nextcloud via the External Folders Feature.

I mount the folders via /etc/fstab entry and use SMB/CIFS. The files paths, Never change. 

This is the second time it happens, that we have accessed memories, and while I see all the albums, they have ZERO pictures in it.
Since files get added externally, I have a periodic cron jobs running Nextcloud files scan, and also memories Index.

`*/10 * * * * docker ps --format "{{.Names}}" | grep -q "^nextcloud-aio-nextcloud$" && docker exec --user www-data nextcloud-aio-nextcloud php occ files:scan --all`

`*/15 * * * * docker ps --format "{{.Names}}" | grep -q "^nextcloud-aio-nextcloud$" && docker exec --user www-data nextcloud-aio-nextcloud php occ memories:index`

Everything is connected via lan and 10G switches, however I am guessing, it could be that if there is an intermittent network disconnection memories loses all info, but I can see all photos and videos in the Timeline. 

After a chat with Pulsejet, I am thinking that may be if the `nextcloud files:scan -all` runs when the network is down, then it removes files, and when they get readded the `fileid` changes and therefore I lose all media in Albums?

But the thing is, when i ssh into the server and looked into the folder, it did not give me erros like - stale file handles, etc.

Pulsejets view was that `-all` should not cause entries to disappear. 
Do you think this is normal? will removing -all from files:scan help? Or is there something else i can do?

I don't experience this issue on a Photoprism instance using the same media repository. (Unused for a while since I moved to memories)

Ref chat with Pulsejet:
https://github.com/pulsejet/memories/issues/1287

**adding newly seen error** 
Did not show when accessing empty album in Memories, but shows when accessing in Photos.

*Also, noticed that all face recognition data is gone as well.*
```
<html>
<body>
<!--StartFragment-->
Error | index | ExceptionOCA\Photos\Album\AlbumFile::__construct(): Argument #2 ($name) must be of type string, null given, called in /var/www/html/apps/photos/lib/Album/AlbumMapper.php on line 250 in file '/var/www/html/apps/photos/lib/Album/AlbumFile.php' line 32
-- | -- | --


<!--EndFragment-->
</body>
</html>
```

**Comment by szaimen** [comment, marked as answer]
Hi, please open a thread for this at https://help.nextcloud.com thanks!
